Formula/g/gnu-indent.rb

```diff
@@ -5,6 +5,7 @@ class GnuIndent < Formula
   mirror "https://ftpmirror.gnu.org/indent/indent-2.2.13.tar.gz"
   sha256 "9e64634fc4ce6797b204bcb8897ce14fdd0ab48ca57696f78767c59cae578095"
   license "GPL-3.0-or-later"
+  revision 1
```

Comment:
Let's not bump the revision as this doesn't bring anything new/different to the user. We've stopped rev-bumping for such kind of changes.

Reply:
OK, noted.